models/public/cocosnet/model.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from torch import nn, load, cat
+
+import models.networks as networks
+
+
+def remove_all_spectral_norm(item):
+    if isinstance(item, nn.Module):
+        try:
+            nn.utils.remove_spectral_norm(item)
+        except Exception:
+            pass
+        for child in item.children():
+            remove_all_spectral_norm(child)
+    if isinstance(item, nn.ModuleList):
+        for module in item:
+            remove_all_spectral_norm(module)
+    if isinstance(item, nn.Sequential):
+        modules = item.children()
+        for module in modules:
+            remove_all_spectral_norm(module)
```

Comment:
`Sequential` is a subtype of `Module`, so this case is redundant.

I suspect the `ModuleList` case might be redundant too.